pages/api/openPR.js

```diff
@@ -29,10 +29,22 @@ function parseProjectName(values) {
 }
 // Create Github checkout branch for current submissions
 function createGithubCheckoutBranch(name) {
+  let nameArray = Array.from(name);
+  nameArray = nameArray.filter((character, index, array) => {
+    if (character == "." || character == "~" || character == "^" || character == ":") {
+      console.log();
```

Comment:
@nathanfletcher: I find these `console.log()` statements strange because you are not logging anything, so then they are redundant. I would ask to remove them and do nothing, or better yet combine both if and negate them, so that "if (condition) then return character" would result in a much cleaner code.

Reply:
@lacabra I changed it and made it shorter by using switch cases instead
fixed here [0600e4d](https://github.com/lacabra/submission-digitalpublicgoods/pull/37/commits/0600e4d89c859730deb5c9bf80102b03426d5f3a)